helm.yaml

```diff
@@ -28,6 +28,8 @@ pipeline:
 
   - runs: |
       cd helm
+      go get github.com/docker/distribution@v2.8.2-beta.1
```

Comment:
Can we move to 2.8.2 instead of beta 
 go get github.com/docker/distribution@v2.8.2-beta.1 ->  go get github.com/docker/distribution@v2.8.2